### PR TITLE
[Localization] #1761 Korean trainer dialogue (some unnamed trainers)

### PR DESCRIPTION
**Pull request by sodaMelon**
[Localization] #1761 Korean trainer dialogue (some unnamed trainers)
## What are the changes?

Translating locales\ko\dialogue.ts into Korean.
This PR is added korean dialogue for some unnamed trainers

@sodaMelon @Ohmry @returntoice now participating in https://github.com/pagefaultgames/pokerogue/issues/1761 translating project.

## Why am I doing these changes?

Few trainers's dialogue have been translated into Korean yet.

## What did change?

I translated **some nonamed trainer battles' dialogues.**
Also I checked their appearances that exist in [repository](https://github.com/pagefaultgames/pokerogue/tree/main/public/images)

랜덤이름 언네임드 트레이너 다이얼로그를  몇명 번역했습니다.
(scientist. school_kid, artist, guitarist, worke, rworker_female, worker_double)

말투에는 최근 정발판 스칼렛+레포지토리에 존재하는 트레이너 스프라이트 이미지 참고했습니다.

부자연스럽다면 편하게 Suggestion 주세요!

### Screenshots/Videos

( very hard to meet them.... and it's dosen't matter)

## How to test the changes?

It seems there won't be much to test since it's just a translation of an existing .ts file. You can check it when you meet the respective trainers.

## Checklist
- [x] There is no overlap with another PR?
- [x] The PR is self-contained and cannot be split into smaller PRs?
- [x] Have I provided a clear explanation of the changes?
- [x] Have I tested the changes (manually)?
    - [x] Are all unit tests still passing? (`npm run test`)
- ~~[ ] Are the changes visual?~~
  - ~~[ ] Have I provided screenshots/videos of the changes?~~

### CODE REVIEW

File: src/locales/ko/dialogue.ts [hunk 1]
```diff
@@ -243,64 +243,64 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
     },
     "defeat": {
-      1: "My body and mind aren't necessarily always in sync."
+      1: "생각처럼 몸이 잘 안따라주네."
     }
   },
   "worker_double": {
     "encounter": {
-      1: "I'll show you we can break you. We've been training in the field!",
+      1: "어떻게 널 부실 수 있는지 보여줄게. 우린 현장에서 수련해왔거든!",
```
Comment (GINK-SS) [inline src/locales/ko/dialogue.ts:300]:
```suggestion
      1: "어떻게 널 부실 수 있는지 보여줄게. 우린 현장에서 수련해 왔거든!",
```

File: src/locales/ko/dialogue.ts [hunk 2]
```diff
@@ -243,64 +243,64 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
     },
     "victory": {
-      1: "How strange… How could this be… I shouldn't have been outmuscled.",
+      1: "이상하네… 어떻게 이럴 수 있지… 힘으로 압도할수 없다니.",
```
Comment (GINK-SS) [inline src/locales/ko/dialogue.ts:303]:
```suggestion
      1: "이상하네… 어떻게 이럴 수 있지… 힘으로 압도할 수 없다니.",
```

File: src/locales/ko/dialogue.ts [hunk 3]
```diff
@@ -243,64 +243,64 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
+      1: `사람들이 나를 오해하는 게 신경 쓰여.
+                $나는 그것보다 훨씬 깨끗한데.`
     },
     "victory": {
-      1: "I really don't want my skin to burn, so I want to stay in the shade while I work."
+      1: "피부가 타는 게 싫어서, 일하는 동안엔 그늘에 머물고 싶어."
     },
     "defeat": {
-      1: "My body and mind aren't necessarily always in sync."
+      1: "생각처럼 몸이 잘 안따라주네."
```
Comment (GINK-SS) [inline src/locales/ko/dialogue.ts:295]: 이 번역 부분 진짜 맘에 드네요 :)

File: src/locales/ko/dialogue.ts (hunk 1 above)
Comment (goldilocks25) [inline src/locales/ko/dialogue.ts:300]: 너를 무너뜨릴 수 있다는 것을 보여줄게. 우리는 실전 경험이 있거든!

File: src/locales/ko/dialogue.ts [hunk 4]
```diff
@@ -243,64 +243,64 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
     },
     "victory": {
-      1: "Silenced for now, but my melody of resilience will play on.",
+      1: "지금은 조용하지만, 회복의 멜로디를 연주할 거야.",
     },
   },
   "worker": {
     "encounter": {
-      1: "It bothers me that people always misunderstand me. I'm a lot more pure than everyone thinks.",
+      1: "사람들이 저를 오해하는 게 신경 쓰여요. 저는 그것보다 훨씬 깨끗합니다.",
```
Comment (returntoice) [inline src/locales/ko/dialogue.ts:280]:
```suggestion
      1: "사람들이 저를 오해하는 게 신경 쓰여요. 전 생각보다 훨씬 깨끗하답니다.",
```
'그것보다'보다는 '생각보다'라고 하면 의미가 좀 더 명확해지는 듯합니다.

File: src/locales/ko/dialogue.ts [hunk 5]
```diff
@@ -243,64 +243,64 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
-      1: "I really don't want my skin to burn, so I want to stay in the shade while I work.",
+      1: "피부가 타는 게 싫어서, 일하는 동안엔 그늘에 머물고 싶어요.",
     },
   },
   "worker_female": {
     "encounter": {
-      1: `It bothers me that people always misunderstand me. 
-                $I'm a lot more pure than everyone thinks.`
+      1: `사람들이 나를 오해하는 게 신경 쓰여.
+                $나는 그것보다 훨씬 깨끗한데.`
```
Comment (returntoice) [inline src/locales/ko/dialogue.ts:289]:
```suggestion
                $나는 생각보다 훨씬 깨끗한데.`
```

File: src/locales/ko/dialogue.ts (hunk 1 above)
Comment (returntoice) [reply src/locales/ko/dialogue.ts:300]: '부시다'는 '설거지하다' 또는 '눈이 부시다' 같은 의미로만 사용되는 단어여서, '부술'이라고 하거나 @goldilocks25님처럼 번역하는 것이 더 좋을 듯합니다.

**Review by returntoice — CHANGES_REQUESTED** [review]

File: src/locales/ko/dialogue.ts (hunk 1 above)
Comment (sodaMelon) [reply src/locales/ko/dialogue.ts:300]: > 너를 무너뜨릴 수 있다는 것을 보여줄게. 우리는 실전 경험이 있거든!

채택하겠습니다 감사합니다

Comment (sodaMelon) [reply src/locales/ko/dialogue.ts:300]:
```suggestion
      1: "너를 무너뜨릴 수 있다는 것을 보여줄게. 우리는 실전 경험이 있거든!",
```
트레이너 이미지가 공사장 일꾼이더라구요 ㅋㅋ 거기 착안하다가 놓쳤는데 변경하겠습니다

**Comment by sodaMelon** [comment]
Thanks to all suggestions! I fixed dialogues more beautiful.

Please check again 🤗 when you have free-time.

**Review by returntoice — APPROVED** [review]

**Review by XGSleepWalker — APPROVED** [review]

**Pull request merged**